### PR TITLE
ssl check script + add redis to requirement.txt

### PR DESCRIPTION
**Pull request by yufangzhang**
this file was missing by some magic.

### CODE REVIEW

File: export_data_to_elk.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import sys
+import json
```
Comment (cliffxuan) [inline export_data_to_elk.py]: there should be a blank line here between `json` and `redis` per pep8
```
Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.
```

File: export_data_to_elk.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    else:
+        return False
+
+
+def connect_to_redis():
+    try:
+        r = redis.StrictRedis('localhost', 16379, 0)
+    except (redis.exceptions.ConnectionError,
+            redis.exceptions.BusyLoadingError) as e:
+        print("Redis connection error: %s" % e,
```
Comment (cliffxuan) [inline export_data_to_elk.py]: i think `"Redis connection error: {} {}".format(e, sys.stedrr)` is the preferred way for string formatting.

File: export_data_to_elk.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    except ValueError as e:
+        print ("Read json file error: %s" % e, sys.stderr)
+        sys.exit(1)
+    except:
+        print "Unexpected error when reading sys.stdin"
+        sys.exit(1)
+    return ssl_logs
+
+
+def parseLog(ssl_logs, r):
```
Comment (cliffxuan) [inline export_data_to_elk.py]: should be `parse_log`

File: export_data_to_elk.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+
+def connect_to_redis():
+    try:
+        r = redis.StrictRedis('localhost', 16379, 0)
+    except (redis.exceptions.ConnectionError,
+            redis.exceptions.BusyLoadingError) as e:
+        print("Redis connection error: %s" % e,
+              sys.stderr)
+        sys.exit(1)
```
Comment (cliffxuan) [inline export_data_to_elk.py]: we probably should avoid using `sys.exit`. re-raising the exception is more pythonic.

File: reports/ssl_check/export_data_to_elk.py [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+    e.g. append "endpoints"."details"."vulnBeast" to the the same hierarchy as "endpoint".
+    Args:
+        host(string): "vulnBeast" as mentioned above for example
+        tag(dict): [json_file]["endpoint"]["details"] in the above example.
+        host(dict): one host(json obj) from the ssl_check json array file
+    """
+    if ele in tag:
+        extracted = tag[ele]
+        host[ele] = extracted
```
Comment (cliffxuan) [inline reports/ssl_check/export_data_to_elk.py]: this line brings in side effect, which can be unsafe given the name of the function suggests it's side effect free. there must be a better way to do it.

File: export_data_to_elk.py (hunk 1 above)
Comment (yufangzhang) [inline export_data_to_elk.py]: Thanks for the comments guys! I made a mistake with the commits, need to create a new PR.

File: export_data_to_elk.py (hunk 2 above)
Comment (yufangzhang) [inline export_data_to_elk.py]: Thanks for the comments guys! I made a mistake with the commits, need to create a new PR.

**Comment by cliffxuan** [comment]
feel free to ignore my comments. it's more for application code rather than system code. :)

**Pull request closed without merging**